XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -42,6 +38,78 @@ private void InitializeData()
+            nestedCategories.Add(
+                new NestedCategory("Fruits",  new ObservableCollection<string>{
+                                                            "Apricots",
+                                                            "Bananas",
+                                                            "Grapes",
+                                                            "Strawberries",
+                                                            "Watermelon",
+                                                            "Plums",
+                                                            "Blueberries"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Vegetables", new ObservableCollection<string>{
+                                                            "Broccoli",
+                                                            "Spinach",
+                                                            "Sweet potato",
+                                                            "Cauliflower",
+                                                            "Onion",
+                                                            "Brussel sprouts",
+                                                            "Carrots"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Grains", new ObservableCollection<string>{
+                                                            "Rice",
+                                                            "Quinoa",
+                                                            "Pasta",
+                                                            "Bread",
+                                                            "Farro",
+                                                            "Oats",
+                                                            "Barley"
+                }));
+
+            nestedCategories.Add(
+                new NestedCategory("Proteins", new ObservableCollection<string>{
+                                                            "Steak",
+                                                            "Chicken",
+                                                            "Tofu",
+                                                            "Salmon",
+                                                            "Pork",
+                                                            "Chickpeas",
+                                                            "Eggs"
+                }));
+
+            outerRepeater.ItemsSource = nestedCategories;
+
+            // Set sample code to display on page's initial load
+            SampleCodeLayout.Value = "<muxc:StackLayout x:Name='VerticalStackLayout' Orientation='Vertical' Spacing='8'/>";
+
+            SampleCodeDT.Value = @"<DataTemplate x:Key=""HorizontalBarTemplate"" x:DataType=""l: Bar"">
```

Comment:
```suggestion
            SampleCodeDT.Value = @"<DataTemplate x:Key=""HorizontalBarTemplate"" x:DataType=""l:Bar"">
```

I believe the extra space would prevent that code from working.